src/main/java/fi/aalto/cs/apluscourses/model/Course.java

```diff
@@ -59,12 +63,14 @@
    * @param resourceUrls    A map containing URLs to resources related to the course. The keys are
    *                        the names of the resources and the values are the URLs.
    */
-  public Course(@NotNull String name,
+  public Course(@NotNull String id,
```

Comment:
Should we use ID also in `CourseProjectViewModel` when solving value for `currentSettingsDiffer`?

Reply:
Yes we absolutely should, good catch :+1: